fxmanifest.lua

```diff
@@ -5,29 +5,30 @@ description 'https://github.com/Qbox-project'
 version '1.0.0'
 
 shared_scripts {
-	'@qbx-core/shared/locale.lua',
-	'@qbx-core/import.lua',
-	'locales/en.lua',
-	'locales/*.lua',
-	'config.lua',
-	'@ox_lib/init.lua',
+    '@qbx-core/shared/locale.lua',
+    '@qbx-core/import.lua',
+    'locales/en.lua',
+    'locales/*.lua',
+    'config.lua',
+    '@ox_lib/init.lua',
 }
 
 server_scripts {
-	'server/main.lua',
+    'server/main.lua',
 }
 
 client_scripts {
-	'client/**/*.lua'
+    'client/**/*.lua',
+    'load-unload.lua'
```

Comment:
Oh. This is just in the wrong place. Move load-unload.lua into the client folder instead of changing the fxmanifest.lua :)